packages/react/src/components/internal/TextFieldWrapper/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import classNames from 'classnames';
+
+export interface TextFieldWrapperProps extends React.HTMLAttributes<HTMLDivElement> {
+  children: React.ReactNode;
+  className?: string;
+}
+
+const TextFieldWrapper = ({
```

Comment:
This component is just missing `ref`, see [component standards](https://cauldron.dequelabs.com/component-guidelines#component-standards)